sdk/core/amqp-common/changelog.md

```diff
@@ -1,3 +1,8 @@
+### TBD 1.0.0-preview.16
```

Comment:
Since we have no other changes planned for amqp-common, let's add the release date as part of this PR

Reply:
Unless we plan to port changes from #9503, #9494 etc

Reply:
Looks like #9503 is not needed because of 
![image](https://user-images.githubusercontent.com/10452642/90449164-c0f42580-e09b-11ea-9072-4ef6ef2c84ab.png)

Reply:
I'll get #9494 too. That sounds like an annoying bug.

Reply:
Added in!

@chradek, even when you're not directly on a PR....you're fixing bugs. Amazing. ;)